collections/ansible_collections/cloudkit/templates/roles/ocp_4_17_small/tasks/delete_template.yaml

```diff
@@ -0,0 +1,19 @@
+- ansible.builtin.import_role:
+    name: hosted_cluster
```

Comment:
I find it confusing to reference a top-level role from a collection, we should probably move these roles in a dedicated collection (or it the template collection directly?).

Reply:
I don't think they should live in the template collection, but I can see moving them into a separate collection. Let's plan on that in a subsequent pr.